### PR TITLE
Upgrade govuk elements/toolkit/template

### PR DESCRIPTION
**Pull request by Guntrisoft**
This branch updates the following dependencies on the pension_guidance project:

* https://github.com/alphagov/govuk_template - to 0.18.0
* https://github.com/alphagov/govuk_frontend_toolkit to 4.14.1
* https://github.com/alphagov/govuk_elements to 1.2.2

### CODE REVIEW

**Comment by Guntrisoft** [comment]
@benbarnett

**Comment by benbarnett** [comment]
Had to do a fairly hefty rewrite on the git history but the majority of this work was done by Mr @Guntrisoft!

File: content/cookies.md [hunk 1]
```diff
@@ -19,8 +19,8 @@ Find out more about [how to manage cookies](http://www.aboutcookies.org).
 We use Google Analytics software to collect information about how you use our website. We do this to help make sure the website is meeting the needs of its users and to help us make improvements.
 Google Analytics stores information about:
 
-- the pages you visit on the Pension Wise website  - how long you spend on each page 
-- how you got to the website  
+- the pages you visit on the Pension Wise website - how long you spend on each page
```
Comment (dwhenry) [inline content/cookies.md]: This feels like a mistake. looks like `how long you spend on each page` should be on it's own line.

Comment (benbarnett) [inline content/cookies.md]: Good spot!

**Comment by Guntrisoft** [comment]
👍 outstanding

**Pull request merged**